.github/workflows/update-latest-images.yml

```diff
@@ -62,5 +66,6 @@ jobs:
         run: |
           git config --global user.name "Mykhailo Kuznietsov"
           git config --global user.email "mkuznets@redhat.com"
-          export GITHUB_TOKEN=${CHE_INCUBATOR_BOT_GITHUB_TOKEN}
-          ./updateBaseImages.sh -b main -maxdepth 5 -f *Dockerfile ${{ github.event.inputs.params }}
+          export GITHUB_TOKEN=${CHE_BOT_GITHUB_TOKEN}
+
+          /tmp/updateBaseImages.sh -b master -maxdepth 3 -f *Dockerfile ${{ github.event.inputs.params }}
```

Comment:
could branch be a parameter so we can run this on 7.yy.x branch too?

Reply:
added parameter `branch`